app/renderer/components/preferences/payment/enabledContent.js

```diff
@@ -274,6 +289,18 @@ const gridStyles = StyleSheet.create({
   }
 })
 
+const loaderAnimation = {
```

Comment:
I know still WIP but please move this to `animations.js` file which is avail in 0.19.x too

Reply:
done